ee/tabby-db/src/github_repository_provider.rs

```diff
@@ -0,0 +1,50 @@
+use anyhow::{anyhow, Result};
+use sqlx::{query, query_as};
+
+use crate::{DbConn, SQLXResultExt};
+
+pub struct GithubProviderDAO {
```

Comment:
as said, all namings should be updated. in this case `GithubRepositoryProviderDAO`.

Similar to many places below